apps/src/blockly/blockSvgUnused.js

```diff
@@ -0,0 +1,213 @@
+import msg from '@cdo/locale';
```

Comment:
Most of this file is copied wholesale from https://github.com/code-dot-org/blockly/blob/main/core/ui/block_svg/block_svg_unused.js, so I'll add some inline comments of relevant changes